libflux/src/core/ast/flatbuffers/ast_generated.rs

```diff
@@ -1203,6 +1216,269 @@ pub mod fbast {
         }
     }
 
+    pub enum DictTypeOffset {}
+    #[derive(Copy, Clone, Debug, PartialEq)]
+
+    pub struct DictType<'a> {
+        pub _tab: flatbuffers::Table<'a>,
+    }
+
+    impl<'a> flatbuffers::Follow<'a> for DictType<'a> {
+        type Inner = DictType<'a>;
+        #[inline]
+        fn follow(buf: &'a [u8], loc: usize) -> Self::Inner {
+            Self {
+                _tab: flatbuffers::Table { buf: buf, loc: loc },
+            }
+        }
+    }
+
+    impl<'a> DictType<'a> {
+        #[inline]
+        pub fn init_from_table(table: flatbuffers::Table<'a>) -> Self {
+            DictType { _tab: table }
+        }
+        #[allow(unused_mut)]
```

Comment:
This code is generated by `flatc`. My guess is that in certain cases this generated variable is unused and because this is rust, it wouldn't compile without this annotation.